playground/infrastructure/cloudbuild/Dockerfile-base-image-worker

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@ruslan-ikhsan I propose to name this file just Dockerfile this will simplify the commands to build it and run it